JSON/Alfa-Bravo.json

```diff
@@ -1,6 +1,7 @@
 {
   "Author": "NickLatkovich",
```

Comment:
![image](https://user-images.githubusercontent.com/56419667/114229776-86464280-9935-11eb-8b05-2f325565dfa3.png)
Are you sure this how you intend the credits to look on repo? Grunkle will not be displayed on the page unless you hover mouse cursor on top of your name. If this is what you intend, ignore this message. If not, please delete the author field.

Reply:
Side not, if you have Contributors field in JSON file, you do not need an Author field. Repo will merge all contributors from Manual Writers and Developers and display that automatically.